sale_restricted_qty/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Akretion (<http://www.akretion.com>)
+# Copyright 2024 CorporateHub
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale order min quantity",
```

Comment:
```suggestion
    "name": "Sale order restrict quantity",
```
This module seems to do more then only min quantity.

Reply:
These are addressed in #2997